src/status_im/contexts/settings/keycard/view.cljs

```diff
@@ -2,16 +2,53 @@
   (:require [quo.core :as quo]
             [quo.foundations.colors :as colors]
             [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
             [status-im.common.events-helper :as events-helper]
             [status-im.common.resources :as resources]
             [status-im.constants :as constants]
+            [status-im.contexts.settings.keycard.style :as style]
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
-(defn view
+(defn registered-keycard
+  [{:keys [profile-name profile-image customization-color keycard-name]}]
+  [rn/view {:style style/keycard-row}
+   [quo/icon :i/keycard-card
+    {:size  20
+     :color colors/white-70-blur}]
+   [rn/view {}
```

Comment:
`[:<>`

Reply:
@flexsurfer Since the `style/keycard-row` has `:gap`, removing  the `view` will modify the styles.

Not all non-styled  `view` components must be removed